src/store/ChartState.js

```diff
@@ -253,6 +253,8 @@ class ChartState {
     }
 
     @action.bound setOnPagination({ end }) {
+        // TODO: remove next line when the fix of markers is done
```

Comment:
@kellybinary this is the temporary fix that needs to be removed later

Reply:
this fixes the missing chart loader, thanks @nazaninreihani  🎉